tests/test_timeit_logger.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import print_function
+import logging
+import unittest
+from mock import patch
+import profilehooks
+import sys
+import time
```

Comment:
Import ordering: let's use the pep-8 suggested grouping:

1. future imports
2. stdlib imports
3. 3rd-party librarie s(i.e. mock)
4. local code (i.e. profilehooks)

with blank lines between sections.